index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio Mobile Version</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> -->
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+       <main>
+         <div class="wrapper">
+               <!-- the toolbar or header section -->
+        <header>
+            <h4>My Logo</h4>
+           <!-- <img class="img" src="ham.png" alt="hambuger menu"> -->
+           <span class="hamb">&#9776;</span>
+       </header>
+
+        <!-- headline section -->
+       <section class="headline-section">
+           <div class="back-drop">
+            <img class="eclipse-11" src="/images/Ellipse 11.png">
+            <img class="rect-7" src="/images/Rectangle 7.png">
+            <img class="eclipse-5" src="/images/Ellipse 5.png">
+            <img class="rect-4" src="/images/Rectangle 4.png">
+            <img class="eclipse-3" src="/images/Ellipse 3.png">
+           </div>
+          <h1>I'm Saningo Glad to see you</h1>
+          <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need done, dont hestitate to contact me.</p>
+       </section>
+       <section class="social-details">
+           <h5>LET'S CONNECT</h5>
+           <div class="social-wrapper">
+            <span class="social-icon"><img src="/images/Vector.svg"></span>
+            <!-- <span class="social-icon linkedin"><img src="/images/Vector.png"></span> -->
+            <span class="social-icon"><img src="/images/Vector.png"><img src="/images/Vector (1).png"></span>
+            <span class="social-icon"><img src="/images/Vector (1).svg"></span>
+            <span class="social-icon"><img src="images/Vector (2).svg"></span>
+            <span class="social-icon"><img src="images/Vector (3).svg"></span>
+           </div>
+       </section>
+       <section>
+           <span><img class="eclipse-12" src="/images/Ellipse 12.png"></span>
+           <span><img class="eclipse-13" src="/images/Ellipse 13.png"></span>
+           <span><img class="eclipse-15" src="/images/Ellipse 15.png"></span>
+           <span><img class="rect-5" src="/images/Rectangle 5.png"></span>
+           <span><img class="eclipse-4" src="/images/Ellipse 4.png"></span>
```

Comment:
- [ ] These images are for style purposes, I would recommend using the `background-image` property in the CSS since they are just for styling, not content images. I went ahead and recorded a Loom video to help you understand how to get a background image from Figma design will make it easier for you
[Loom Video](https://www.loom.com/share/8bb9334592b941068a569955e395c441)